lld/ELF/Target.cpp

```diff
@@ -45,6 +45,15 @@ std::string lld::toString(RelType type) {
   return std::string(s);
 }
 
+const ELFSyncStream &elf::operator<<(const ELFSyncStream &s, RelType type) {
+  StringRef buf = getELFRelocationTypeName(s.ctx.arg.emachine, type);
+  if (buf == "Unknown")
+    s << ("Unknown (" + Twine(type) + ")").str();
```

Comment:
```suggestion
    s << "Unknown (" << type << ')';
```

Reply:
It seems like the suggestion wasn't taken. Does this not work?